package.json

```diff
@@ -25,7 +25,7 @@
   "devDependencies": {
     "@cosmjs/stargate": "^0.28.9",
     "@subql/cli": "^5.4.0",
-    "@subql/node-cosmos": "^4.2.1",
+    "@subql/node-cosmos": "4.2.1",
```

Comment:
please retain semver range. the lockfile controls whether the update actually changes

Heads up that a new release is imminent with this dependency updates (`https://github.com/subquery/subql-cosmos/pull/302`) fixing the exit code for the test runner.